### DISCUSSION TITLE
Default Problem Request Template is missing in settings

### DISCUSSION THREAD

**Discussion by lukep-axim** [AdhocAdam/smletsexchangeconnector#490] [Q&A]
Good day, 

When trying to setup the SMLets Exchange connector, it won't continue due to the problem template being missing. I tried setting up a template for it however not sure how to get one to display in the dropdown. I really need to get this up and running as soon as possible if I could get any assistance, please. 

Thanks

![image](https://github.com/AdhocAdam/smletsexchangeconnector/assets/174775579/0d80c68e-0f33-46c9-aa1e-8830875ad680)

**Comment by AdhocAdam** [comment]
It's not entirely clear to me how this could be happening based on how the connector's UI is built as there is no filtering in the code. It's simply loading all of the available Templates of a given Work Item class for a specific drop down.

https://github.com/AdhocAdam/smletsexchangeconnector/blob/02be0b1aed028647848c982e19ae5779c7596822/ManagementPack/2016/SMLetsExchangeConnectorSettingsUI/AdminSettingsWizardData.cs#L2892-L2909

Given this, there are really only two things that come to mind here if you're not seeing a Template you expect:
- Is the PR Template you've built, based off of a custom inherited Problem class?
- Is the PR Template you've defined stored in a Microsoft unsealed MP or your own custom unsealed MP?
